### PR TITLE
Docker images config from GitHub

### LINKED ISSUE #507: BN docker image version from github
At the current state, we use the docker image version from env.
We want to change it to pull the image version from this repository to simplify updating versions of images if there are no breaking changes for the chain guardian

### PR DESCRIPTION
**Pull request by BeroBurny**
**Short description of work done**

- implement config reader from github

### PR Checklist

- [x] I have run type check locally
- [x] I have run linter locally
- [ ] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master

### Changes

- implement config reader from github
- refactor sagas to use docker image from config

### Issues

Closes #507

### CODE REVIEW

File: docker.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "images": {
+    "lighthouse": {
+      "current": "sigp/lighthouse:v1.1.3",
```
Comment (mpetrunic) [inline docker.json:4]: Can we just hardcode "recommended" version but display all versions from github in dropdown?

Comment (mpetrunic) [reply docker.json:4]: Fetch from this endpoint https://api.github.com/repos/nodefactoryio/chainguardian/releases (chaneg repo and owner to lighthouse)

Comment (BeroBurny) [reply docker.json:4]: think it's better to keep it like this is for now and discuss this in the next weekly?

**Comment by mpetrunic** [comment]
Added clarification to issue:
https://github.com/NodeFactoryIo/ChainGuardian/issues/507

**Comment by BeroBurny** [comment]
image

**Comment by mpetrunic** [comment]
Can we put some badge next to recommended version in dropdown?

**Comment by BeroBurny** [comment]
> Can we put some badge next to recommended version in dropdown?

something like this? 
image

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**